doc/source/guidelines/doc_practices.rst

```diff
@@ -0,0 +1,352 @@
+specialists may not know how to do this. You can easily use either Windows PowerShell
+commands or GitBash.
+
+Using Windows PowerShell to Clone a GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. In Windows PowerShell, navigate to where you clone repositories on your machine.
+
+#. Go to the `PyAnsys GitHub account <https://github.com/pyansys>`_ and select the
+   repository that you want to clone.
+
+   Because the list of repositories is quite long, you might want to use the ``Find
+   a repository`` option to search for the repository that you want to clone. For
+   example, you can seach for "theme" to find the ``pyansys-sphinx-theme`` repository.  
+
+#. On the right, click the green ``Code`` button to drop down a list of options.
+
+#. To the right of the HTTPS entry, click the clipboard icon to copy the path to
+   the repository.
+ 
+#. Go back to Windows Powershell and type ``git clone`` and then click the right
+   mouse button to paste the copied path.
+
+#. Press ``Enter`` to copy the files in the repository to yoour local directory.
+
+#. Type ``ls`` to see a list of the files now in your local directory. 
+
+.. note::
+   In Anaconda Powershell, typing ``dir`` is the equivalent to typing ``ls``.
+
+Using GitBash to Clone a GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. In the folder where you clone PyAnsys repositories, right-click and select
+   ``GitBash Here``.
+
+#. Type ``git clone`` followed by the address of the repo to clone.
+
+For example, to clone the ``pyansys-sphinx-theme`` repository, you would
+enter:
+  
+.. code::
+  
+  git clone https://github.com/pyansys/pyansys-sphinx-theme
+
+.. _create_pr:
+
+Pushing Changes to the GitHub Repository
+----------------------------------------
+Once you have tested your changes in local documentation builds and are
+satisifed with them, you can push them to the library's GitHub repository. 
+You can use Git commands or Git Extension to accomplish this. 
+
+For documentation changes, the branch name should have a prefix of ``doc\``
+followed by a short description of what you've changed. For more information, see
+:ref:`branch_naming`.
+
+Using Git Commands to Push Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This procedure assumes that you are using Git Commands to push your changes to the
+GitHub repository.
+
+#. In Windows Command Prompt, navigate to the directory where you cloned 
+   the GitHub reposity.
+
+#. Type ``git status`` to see a summary of the changes that you have made in this
+   directory.
+
+#. Create a branch by entering ``git checkout -b`` followed by the name to give
+   this branch.
+   
+   The option ``-b`` is for adding a branch. If you were adding a branch with documentation
+   changes to the ``Contributing`` topic, you might enter:
+   
+   .. code::
+
+    git checkout -b edit_contributing
+   
+#. To add and check out this new branch, enter ``git add checkout -b doc/`` followed by
```

Comment:
This command is incorrect, it's either `add` or `checkout -b`. Since we `checkout` before and `add` after this, I think this bullet point can just be removed.